### PR TITLE
Branch UI icons

### PR DESCRIPTION
**Pull request by nyankoclaws**

### CODE REVIEW

File: src/main/resources/view/PersonListPanel.fxml
Comment (PallonCX) [inline src/main/resources/view/PersonListPanel.fxml:1]: Why is the maxWidth added?

**Review by JustWeiHao — APPROVED** [review]
LGTM

File: src/main/resources/view/PersonListPanel.fxml
Comment (nyankoclaws) [reply src/main/resources/view/PersonListPanel.fxml:1]: Attempt to allow the width of the personCard and the respective list panel to be a higher width for the icons to go further right. Can remove if dw but might be useful if someone else is using the styling.

**Review by donwong2308 — APPROVED** [review]
LGTM

**Review by PallonCX — APPROVED** [review]
LGTM

**Review by larainezo — APPROVED** [review]
LGTM

**Pull request merged**